NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# User can view post comments on post details

#### Changes Made
1. Added Comment.js
1. Modified CommentController.cs, CommentRepository.cs/ICommentRepository.cs, PostDetails.js, and commentManager.js
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jf-viewComments
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Log in as an authenticated user, and navigate to a post's detail page
    > The post detail page shoulder render all of the post's comments at the bottom of the page
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
issue #28
